pkg/capabilities/consensus/ocr3/identical_consensus.go

```diff
@@ -0,0 +1,38 @@
+package ocr3
+
+import (
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink-common/pkg/workflows"
+)
+
+// Note this isn't generated because generics isn't supported in json schema
+
+type IdenticalConsensusConfig[T any] struct {
+	Encoder       Encoder
+	EncoderConfig EncoderConfig
+}
```

Comment:
I don't think I understand. Why does an identical consensus configuration have an encoder in it? I thought this would be an aggregation method and config.

Reply:
@nolag I think I agree with Deividas here: the aggregator + the encoder are two different parts of the consensus API.

Reply:
I don't follow the "and config" this is the config, config is in the name.

Reply:
After sync with @cedric-cordenier, the best would be to have an embedded struct with these two and report id.  Right now, I'm not sure how to get the json schema generator to embed a struct, so I'm going to leave them at the top level.